lib/nadia/api.ex

```diff
@@ -59,7 +62,7 @@ defmodule Nadia.API do
   def request(method, options \\ [], file_field \\ nil) do
     timeout = (Keyword.get(options, :timeout, 0) + recv_timeout) * 1000
     method
-    |> build_url
+    |> build_url(options)
```

Comment:
since request url only relates to `method` and `token`, no need to pass all options to `build_url`.

like this:
```elixir
{token, options} = Keyword.pop(options, :token, default_token)

method
|> build_url(token)
...
```